### DISCUSSION TITLE
I can see that dbt-redshift 1.4 was released and published to PyPi. When does the docker image also get published?

### DISCUSSION THREAD

**Discussion by andrassy** [dbt-labs/dbt-redshift#284] [Q&A]
I had assumed that release automation would publish the Python package to PyPi *and* a Docker image to ghcr.io/dbt-labs/dbt-redshift. However, I don't see the Docker image yet even though the release was a couple of days ago. What's the release process for this and how long does that usually take, out of interest? 🙏

**Comment by borismo** [comment]
Still the case at the time of typing this: the latest Docker image tag is 1.5.1 and the latest release is 1.5.6.
In the meantime we pull the 1.5.1 image and run `pip install 1.5.6` which doesn't sound too optimum. Maybe someone has a better solution.
But an up-to-date Docker image would be the best. 🙏
